tests/rules/bem-depth.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+var lint = require('./_lint');
+
+//////////////////////////////
+// SCSS syntax tests
+//////////////////////////////
+describe('bem depth - scss', function () {
+  var file = lint.file('bem-depth.scss');
+
+  // silly case to raise a warning for every selector/placeholder
+  it('[max-depth: 0]', function (done) {
+    lint.test(file, {
+      'bem-depth': [1, { 'max-depth': 0 }]
```

Comment:
Out of curiosity I checked out ESLint and it will let you set the indentation to negative numbers, if you want to. I think this is a case where we let the user be dumb if they want.